app/views/media_objects/_sections.html.erb

```diff
@@ -14,7 +14,7 @@ Unless required by applicable law or agreed to in writing, software distributed
 ---  END LICENSE_HEADER BLOCK  ---
 %>
 <% show_progress = sections.any? { |s| not s.succeeded? } %>
-<% unless (sections.blank? or sections.length == 1) and not show_progress %>
+<% unless (sections.blank? or (sections.length == 1 && sections.first.get_structural_metadata.xpath('//Item').empty?)) and not show_progress %>
```

Comment:
This looks like a scary long line for in a view partial.  Could this be put in a helper so it could be tested?